src/ansi-c/expr2c.cpp

```diff
@@ -113,7 +113,7 @@ static std::string clean_identifier(const irep_idt &id)
 void expr2ct::get_shorthands(const exprt &expr)
 {
   find_symbols_sett symbols;
-  find_symbols(expr, symbols);
+  find_symbols_or_nexts(expr, symbols);
```

Comment:
Yes and no: ok, this is what the function does, but it's not actually what we want. Or at least not in all places. "next" symbols should be handled transparently in verification back-ends, but language front-ends have no business dealing with them.